src/definitions/modules/search.js

```diff
@@ -436,7 +436,6 @@ $.fn.search = function(parameters) {
           },
           result: function(value, results) {
             var
-              lookupFields = ['title', 'id'],
```

Comment:
If we only merge those with the existing searchfield setting here, then this could be all that's needed and the other lines dont have to be changed (and it's not a breaking change :)
```javascript
lookupFields = ['title', 'id'].concat(settings.searchFields).filter(function(item, i, ar){ return ar.indexOf(item) === i; });,
```